io.github.julian_hochhaus.LG4X_V2.yaml

```diff
@@ -0,0 +1,92 @@
+app-id: io.github.julian_hochhaus.LG4X_V2
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-22.08'
+runtime-version: '5.15-22.08'
+command: main.py
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /include
+  - /share/man
+modules:
+  - name: meson
```

Comment:
why do you need to build meson?

Reply:
SciPy Python library version 1.11.1 need meson >=1.1.0, the Sdk provide 0.63.3.